### PR TITLE
Adding resolver documentation

### PR DESCRIPTION
**Pull request by YashviJain01**
Signed-off-by: Yashvi Jain <yashvi.jain@progress.com>

### :nut_and_bolt: Description: What code changed, and why?
Adding resolver documentation in managed services as well

### :chains: Related Resources

https://chefio.atlassian.net/browse/STALWART-218

### :+1: Definition of Done

Added documentation

### :athletic_shoe: How to Build and Test the Change

Screenshots attached

See the preview changes

### :white_check_mark: Checklist

**All PRs** must tick these:

- [X] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [X] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [X] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [X] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [X] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [X] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [X] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [X] Tests added/updated? (all new code needs new tests)
- [X] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable

https://user-images.githubusercontent.com/91940132/191680485-1c2a71f0-5f01-4843-8c4b-8b98fed751b2.png

https://user-images.githubusercontent.com/91940132/191682076-bd161640-7af4-4c5a-9dc0-bf04899da933.png

### CODE REVIEW

**Comment by YashviJain01** [comment]
image

Resolver document in managed services for postgress

**Comment by YashviJain01** [comment]
image

Resolver section in managed services for opensearch

File: components/docs-chef-io/content/automate/managed_services.md [hunk 1]
```diff
@@ -24,6 +24,41 @@ gh_repo = "automate"
...
+
+Apply the changes:
+
+```bash
+sudo chef-automate config patch config.toml
+````
+
+If you wish to reset to the default configuration or to modify the configuration:
+
+1. Run `chef-automate config show config.toml`.
```
Comment (punitmundra) [inline components/docs-chef-io/content/automate/managed_services.md:58]: chef-automate config show  > config.toml

File: components/docs-chef-io/content/automate/managed_services.md [hunk 2]
```diff
@@ -109,6 +144,40 @@ Navigate to `https://localhost:9200/\_dashboards/` in your web browser. To navig
...
+If both options are set, nameserver IPs takes precedence over the system nameserver entries.
+
+Apply the changes:
+
+```bash
+sudo chef-automate config patch config.toml
+````
+If you wish to reset to the default configuration or to modify the configuration:
+
+1. Run `chef-automate config show config.toml`.
```
Comment (punitmundra) [inline components/docs-chef-io/content/automate/managed_services.md:177]: chef-automate config show > config.toml

**Review by punitmundra — CHANGES_REQUESTED** [review]

**Comment by YashviJain01** [comment]
image

Postgres resolver

**Comment by YashviJain01** [comment]
image 

Opensearch resolver

**Review by punitmundra — APPROVED** [review]

**Review by dishanktiwari2501 — APPROVED** [review]

**Comment by YashviJain01** [comment]
> As per the ticket, the changes should be done as part of https://docs.chef.io/automate/troubleshooting/ as well. But I don't see the changes with the preview link. Please update the JIRA ticket with updated acceptance criteria if these are not required.


Removed from the story, troubleshooting was not required as we couldn't find anything there.

**Pull request merged**